d-mode.el

```diff
@@ -33,9 +33,8 @@
 ;; Boston, MA 02111-1307, USA.
 
 ;;; Usage:
-;; Put these lines in your init file.
+;; Put this line in your init file:
 ;;   (autoload 'd-mode "d-mode" "Major mode for editing D code." t)
-;;   (add-to-list 'auto-mode-alist '("\\.d[i]?\\'" . d-mode))
```

Comment:
I think this is still needed if someone not using `package.el` wants to auto-load `d-mode` when opening `.d` files.